packages/core/saved-objects/core-saved-objects-migration-server-internal/src/zdt/model/stages/init.ts

```diff
@@ -115,16 +114,15 @@ export const init: ModelStage<
         ...state,
         controlState: aliasActions.length ? 'UPDATE_ALIASES' : 'INDEX_STATE_UPDATE_DONE',
         ...commonState,
-        newIndexCreation: false,
       };
     // app version is lower than the index mapping version.
-    // likely a rollback scenario - unsupported for the initial implementation
+    // either a rollback scenario, or an old node rebooting during the cohabitation period
+    // in that case, we simply no-op the expand phase.
     case 'lesser':
       return {
         ...state,
-        controlState: 'FATAL',
-        reason: 'Downgrading model version is currently unsupported',
-        logs,
+        controlState: 'INDEX_STATE_UPDATE_DONE',
+        ...commonState,
```

Comment:
Change of behavior - encountering higher versions of the mappings in the index's meta now just skip the schema expansion phase